tests/README.md

```diff
@@ -423,7 +423,7 @@ the test driver machine.
 
 * Start by making sure you're up to date, and install git and ducktape:
 
-        $ sudo apt-get update && sudo apt-get -y upgrade && sudo apt-get install -y git
+        $ sudo apt-get update && sudo apt-get -y upgrade && sudo apt-get install -y pip git
```

Comment:
I was using `"OS: Ubuntu server is recommended"` as listed above. It did not have `pip` installed by default and this did it for me...